icgcget/clients/commands/command_utils.py

```diff
@@ -0,0 +1,34 @@
+import click
+from ..errors import ApiError
```

Comment:
I did attempt that, but it caused issues attempting to import modules.  I'd have to either move cli down to the base Icgc-get folder or nest both commands and clients in a package folder to get it to work.